src/app/public/modules/data-manager/models/data-view-config.ts

```diff
@@ -67,12 +67,12 @@ export interface SkyDataViewConfig {
   searchEnabled?: boolean;
   /**
    * Specifies placeholder text to display in the search input until users enter search criteria.
-   * See the <a href="https://developer.blackbaud.com/skyux/components/search">search component</a> for the default value.
+   * See the <a href="https://developer.blackbaud.com/skyux-v4/components/search">search component</a> for the default value.
    */
   searchPlaceholderText?: string;
   /**
    * Sets the `expandMode` property on the search box for this view.
-   * See the <a href="https://developer.blackbaud.com/skyux/components/search">search component</a> for valid options and the default value.
+   * See the <a href="https://developer.blackbaud.com/skyux-v4/components/search">search component</a> for valid options and the default value.
```

Comment:
This line is now breaking our "max" line length rule. You may need to hit enter between a word (such as between the `a` tag and before `for`). This shouldn't affect the output but will fix the build.

Reply:
I added a line break where you suggested.